### ISSUE TITLE
Transform meshes in parallel

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/AllGUDMeshGen#20]
if possible, to speed it up

**Comment by SteveTownsend** [comment]
Did meshes and Alt Textures. Time before change:
```
2211 total meshes: found 1555 Loose, 593 in BSA, 63 missing files
Generated 6362 with 980 for Alternate Textures, Patched 3128, Skipped 360, Failed 0
Records analysis: 722 ms
Mesh transformation: 34384 ms
Skeleton patching: 1049 ms
```
Time after change:
```
2211 total meshes: found 1555 Loose, 593 in BSA, 63 missing files
Generated 6362 with 980 for Alternate Textures, Patched 3128, Skipped 360, Failed 0
Records analysis: 626 ms
Mesh transformation: 20465 ms
Skeleton patching: 1021 ms
```

**Comment by SteveTownsend** [comment]
The few that were missing are a separate bug with Alt Textures handling, fixed for v1.10.0
